packages/aws-cdk/lib/api/util/cloudformation/stack-activity-monitor.ts

```diff
@@ -621,6 +621,10 @@ export class CurrentActivityPrinter extends ActivityPrinterBase {
       const color = colorFromStatusActivity(res.event.ResourceStatus);
       const resourceName = res.metadata?.constructPath ?? res.event.LogicalResourceId ?? '';
 
+      if (hasErrorMessage(res.event.ResourceStatus ?? '')) {
+        debugger;
+      }
```

Comment:
Did you mean for this to remain here?

It's okay if so, but then I'd kind of like a comment above to explain what this is here to help diagnose, so it can be removed once this investigation is done-done-done.